src/components/pages/home/section/who-is-using-game-ci-section.tsx

```diff
@@ -5,8 +5,62 @@ import styles from './section.module.scss';
 const WhoIsUsingGameCiSection = () => {
   return (
     <Section className={styles.whoIsUsingGameCiSection}>
-      <h2>Developers that chose us</h2>
-      {/* Todo: 5 columns 3 rows for mobile - 10 columns 4 rows for large screens */}
+      <h1>Developers that chose us</h1>
+      <div className={styles.devsGrid}>
+        <div className={styles.gridItem}>
+          <a href="https://www.madronagames.com/">
+            <img src="/assets/images/devs/madronagames-logo.png" alt="Madrona Games" />
+          </a>
+        </div>
+        <div className={styles.gridItem}>
+          <a href="https://totemastudio.com/">
+            <img src="/assets/images/devs/totemastudio-logo.svg" alt="Totema Studios" />
+          </a>
+        </div>
+        <div className={styles.gridItem}>
+          <a href="/">
+            <img src="/assets/images/devs/" alt="" />
+          </a>
+        </div>
+        <div className={styles.gridItem}>
+          <a href="/">
+            <img src="/assets/images/devs/" alt="" />
+          </a>
+        </div>
+        <div className={styles.gridItem}>
+          <a href="/">
+            <img src="/assets/images/devs/" alt="" />
+          </a>
```

Comment:
I'd expect a flex container with a map around flex items here, so that you don't have to manually fill empty spots.

It would also be nice if the flex container would have a max width, but shrink if there are less items.

Not sure if it's something you like to pick up? If not no problem, I can do it too!

Reply:
a comment that was left there in the beginning asked for 5 columns on small screens and 10 on large, so I just followed that. I added the mapping you've requested and I'll let you do the rest :)